lib/src/specs/annotation.dart

```diff
@@ -10,29 +10,45 @@ import 'package:meta/meta.dart';
 import '../base.dart';
 import '../visitors.dart';
 import 'code.dart';
+import 'expression.dart';
 
 part 'annotation.g.dart';
 
+@Deprecated('Use an Expression node instead. Will be removed in 3.0.0.')
 @immutable
-abstract class Annotation
+abstract class Annotation extends Expression
     implements Built<Annotation, AnnotationBuilder>, Spec {
   factory Annotation([void updates(AnnotationBuilder b)]) = _$Annotation;
 
   Annotation._();
 
   /// Part after the `@` in annotation.
-  Code get code;
+  @override
+  Code get code => codeForAnnotation;
+
+  /// Use [code] instead.
+  ///
+  /// This property exists in order to keep compatibility with having a concrete
```

Comment:
I don't understand this... This looks like an unused field. Why do we need it?

Reply:
Ah this is interesting, I can document it better if needed.

Basically `built_value` only generates code for abstract accessors, and because `Expression` implements `code` it "hides" it from `built_value`, and this effectively becomes an empty class.

So I'm redirecting `code` to `codeForAnnotation`. I can make it private though, I think!

Reply:
Nope, can't make it private (breaks built_value). `@protected` it is.

Reply:
Interesting. Could be worth filing an issue on `built_value` to see if this is a use case we want to support without hacks.